contents/docs/cdp/batch-exports/index.md

```diff
@@ -117,3 +117,11 @@ sequenceDiagram
 
 - Some features of the old export destinations are still being ported over.
   - This includes logs and error reporting.
+ 
+## How can I export person info?
+
+Currently it's not possible, but it's on the roadmap:
+- [person data export](https://github.com/PostHog/posthog/issues/20339).
+- Export person properties on events (`event.person_properties`) with [custom schemas](https://github.com/PostHog/posthog/issues/20367).
+
+Note: Our implementation of person ids on events (`event.person_id`) is eventually consistent, which for batch export destinations means joining event `distinct_id` with the person data export.
```

Comment:
Why is this important? `event.person_id` is included in batch exports now?
```suggestion
> **Note:** Our implementation of person IDs on events (`event.person_id`) is eventually consistent. For batch export destinations, this means it requires joining event `distinct_id` with the person data export.
```

Reply:
Because people will internally will update person_ids (overrides table + squash) and they will never get that update and that will be confusing. 
On the other hand persons data export will send them updates, e.g. a new distinct_id was added here.